### DISCUSSION TITLE
Rebuild subproject automatically after you make automatically(watch)

### DISCUSSION THREAD

**Discussion by happytomatoe** [DavidSouther/computron5k#477] [General]
I mean that there is a listener for specific changes. When you press a button in the editor it automatically rebuild a given subproject. 
@DavidSouther @netalondon Do you use some watch to rebuild project? If yes could you share your configuration?

**Comment by DavidSouther** [comment]
Currently, `npx tsc -w` I think is what we're using. You might need to have that in a terminal for each of `simulator` and `components`. `web` should auto-rebuild with `npm start`.
